api/all/src/main/java/io/opentelemetry/api/logs/LogRecordBuilder.java

```diff
@@ -101,6 +108,36 @@ default LogRecordBuilder setAllAttributes(Attributes attributes) {
   /** Sets an attribute. */
   <T> LogRecordBuilder setAttribute(AttributeKey<T> key, T value);
 
+  /** Sets a string attribute. */
```

Comment:
Can we carry over this javadoc note from the Span API? 

https://github.com/open-telemetry/opentelemetry-java/blob/92b089ab2c06846aea3e2cd4e14cbee71608f715/api/all/src/main/java/io/opentelemetry/api/trace/Span.java#L138-L139

Reply:
That's fine, yeah. I had only tried to keep it consistent with the existing setter...but I'll copy it from Span instead.